voxelbotutils/cogs/utils/interactions/components/action_row.py

```diff
@@ -49,7 +49,7 @@ def to_dict(self):
         return [i.to_dict() for i in self.components]
 
     @classmethod
-    def from_dict(cls, data: list):
+    def from_dict(cls, data: dict):
```

Comment:
Again, is this accurate? MessageComponents is a list of ActionRow objects at best.

Reply:
It's returning it to as-written in the library already: [per here](https://github.com/Voxel-Fox-Ltd/VoxelBotUtils/blob/master/voxelbotutils/cogs/utils/interactions/components/action_row.py#L29)